p256/src/lib.rs

```diff
@@ -107,6 +107,10 @@ impl elliptic_curve::Curve for NistP256 {
     const ORDER: U256 = U256::from_be_hex(ORDER_HEX);
 }
 
+#[cfg(feature = "arithmetic")]
+const NZ_ORDER: elliptic_curve::bigint::NonZero<U256> =
+    elliptic_curve::bigint::NonZero::<U256>::new_unwrap(<NistP256 as elliptic_curve::Curve>::ORDER);
```

Comment:
Oof, I say that and I guess you're doing it everywhere. Let me see if I can just make the switch upstream now.

Reply:
Let's make `Curve::ORDER` into `NonZero`: https://github.com/RustCrypto/traits/pull/1969

I also added some const constructors, though you'll have to pull `crypto-bigint` in via git: https://github.com/RustCrypto/crypto-bigint/pull/928